content/src/Blip.Api.Template/Models/TextRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Models
+{
+    [DataContract]
+    public class TextRequest
+    {
```

Comment:
With the change of the POST methods to GET, this will probably not be used. Remove it